AsyncEnumerable.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
         <id>AsyncEnumerator</id>
-        <version>1.2.0</version>
+        <version>1.2.1</version>
```

Comment:
Was just trying to be helpful :).  All commnets have been resolved and rebased.  Let me know if you still want me to fully revert this commit